randovania/games/fusion/logic_database/Main Deck.txt

```diff
@@ -1406,8 +1406,10 @@ Extra - room_id: [50]
   > Beside Pickup
       Any of the following:
           Can Kill Tough Beam-Resistant Enemy
-          # Dodge Pirates: https://www.youtube.com/watch?v=QAffTKp5seU
-          Space Jump or Combat (Intermediate)
+          Any of the following:
+              # Dodge Pirates: https://www.youtube.com/watch?v=QAffTKp5seU
+              Space Jump or Combat (Intermediate)
+              Damage Boosts (Beginner) and Normal Damage ≥ 158
```

Comment:
this look like there's an extra any block in here

Reply:
Unsure what you mean

Reply:
```
any of
   kill tough enemy
   any of
      space or combat
      dmg and boost
```
shouldn't it look like
```
any of
   kill tough enemy
   space or combat
   dmg and boost
```
 I know they mean the same but just looks odd, might just because comment formatting